client/client-sdk-javascript/package.json

```diff
@@ -19,14 +19,20 @@
     "@types/chai": "^4.1.2",
     "@types/chai-as-promised": "^7.1.0",
     "@types/express": "^4.11.1",
+    "@types/json-stable-stringify": "^1.0.32",
```

Comment:
all of these additions seem to be some merge issue, is the client-sdk-js really using these?
probably more can removed now that the mock-server is completely gone

Reply:
I think that only Express is not used anymore